pages/linux/tracepath.md

```diff
@@ -0,0 +1,28 @@
+# tracepath
+
+> Trace the path to a network host discovering MTU along this path.
+> More information: <https://manned.org/tracepath>.
+
+- A preferred way to trace the path to a host:
+
+`tracepath -p 33434 {{host}}`
+
+- Specify the initial destination port, useful with non-standard firewall settings:
+
+`tracepath -p {{destination_port}} {{host}} `
+
+- Print numerical IP addresses only:
+
+`tracepath -n {{host}}`
+
+- Print both hostnames and numerical IP addresses:
+
+`tracepath -b {{host}}`
+
+- Specify a maximum TTL (number of hops):
+
+`tracepath -m {{max_hops}} {{host}}`
+
+- Specify the initial packet length (defaults to 65535 for IPv4 and 128000 for IPv6):
```

Comment:
Since this got included, shouldn't we also explain ho to use IPv6 (it is with `-6` parameter).

How is the IPv6 integration among other commands overall? I do not feel like beginner users are actively seeking it, but might be mistaken.

There could probably be a configuration option of sort to prefer IPv4 (default, at least for now), IPv6 or both to solve this.

Reply:
Aggreed.

Reply:
I don't think it matters a whole lot. 🤷🏻